updater/importUpdate.go

```diff
@@ -17,10 +17,11 @@ func (asset Asset) importUpdate(updatePath string) error {
 	assetFile := filepath.Join(asset.TargetFolder, asset.AssetName+updateFileExtension)
 	const backUpSuffix = ".old"
 	backUp := fmt.Sprint(assetFile, backUpSuffix)
-	err := os.Remove(backUp)
-	printErrors(err)
-	err = os.Rename(assetFile, backUp)
-	printErrors(err)
+	_ = os.Remove(backUp)
+	err := os.Rename(assetFile, backUp)
```

Comment:
Use more compact syntax

```
if e, ok := err.(*QueryError); ok && e.Err == ErrPermission {
    // query failed because of a permission problem
}
```